contracts/manifold/lazyclaim/ERC721LazyPayableClaim.sol

```diff
@@ -151,6 +162,23 @@ contract ERC721LazyPayableClaim is IERC165, IERC721LazyPayableClaim, ICreatorExt
         return _getClaim(creatorContractAddress, claimIndex);
     }
 
+    /**
+     * See {ILazyPayableClaim-getClaimForToken}.
+     */
+    function getClaimForToken(address creatorContractAddress, uint256 tokenId) public override view returns(Claim memory) {
+        TokenClaim memory tokenClaim = _tokenClaims[creatorContractAddress][tokenId];
+        uint256 claimIndex;
+        if (tokenClaim.claimIndex == 0) {
```

Comment:
Why not just check the version since this is a view? Seems like a security hole where someone with an "old" contract could put whatever they want in tokenData and reference a claim they don't own